build.gradle

```diff
@@ -4,12 +4,11 @@ buildscript {
 		jcenter()
 		mavenCentral()
 		maven { url "https://jitpack.io" }
-		maven { url 'https://maven.fabric.io/public' }
+		maven { url  "http://dl.bintray.com/countly/maven" }
```

Comment:
This may or may not cause f-droid issues; see https://github.com/ooni/probe-android/issues/328#issue-623974244.

Isn't it possible to fetch count.ly from bintray's JCenter directly (like we did for probe-engine a while ago)?

Reply:
Don't think so https://support.count.ly/hc/en-us/articles/360037754031-Android-SDK#adding-the-countly-sdk

Reply:
Thanks for sharing the documentation! So, it appears to says the following:

> ### Adding via Android Studio
>
> You may use the default JCenter repository to download the SDK package. If it is not included in your project, include it as follows:
>
> ```Groovy
> buildscript {
>    repositories {
>        jcenter()
>    }
> }
> ```
>
> You can also add the Bintray Maven repository:

My reading of the above is that we probably don't need to manually add their Maven repository and it suffices to fetch their SDK through JCenter that IIRC we are already using. Could you perhaps try and remove their explicit Maven repo and check whether we are able to use Countly anyway through JCenter?

Reply:
edited and works